Add Closure Metadata Tables

thanhz opened the pull request.

TomJKing reviewed (commented):
Looks good. Just wondering whether the table names need the `V2` appended to the end? Not sure it makes sense? Any versioning of the tables will be handled by the migrations, if they ever need changing.

TomJKing commented on lambda/src/main/resources/db/migration/V46__create_file_property_v2.sql:
```diff
@@ -0,0 +1,50 @@
+
+INSERT INTO "FilePropertyV2" ("Name","Description","Fullname","CreatedDatetime","ModifiedDatetime","UserId","PropertyType","Datatype","Editable","MutliValue","PropertyGroup")
+VALUES
+    ('ClientSideOriginalFilepath','The original filepath','Filepath','2021-02-11 08:07:18.000 +0000',NULL,cast('eed88948-daf0-49c6-b511-c61dc6930433' as uuid),'System','text',false,false,NULL),
```
The date's represent the `CreatedDatetime`. Should we not be generating these when we add the tables, instead of using an old date?

thanhz commented:
> Looks good. Just wondering whether the table names need the `V2` appended to the end? Not sure it makes sense? Any versioning of the tables will be handled by the migrations, if they ever need changing.

I have removed v2 from the table name and altered the existing FileProperty table with the new columns and updated any existing data.
